src/combat.ts

```diff
@@ -558,6 +559,18 @@ export function main(): void {
     Macro.if_($monster`Eldritch Tentacle`, Macro.basicCombat())
       .step(Macro.load())
       .submit();
+  } else if (getTodaysHolidayWanderers().length !== 0) {
```

Comment:
It's possible to have both Eldritch Attunement and for holiday wanderers to be available.